bin/docker-compose

```diff
@@ -66,6 +66,7 @@ function __main__() {
   if [[ "${SERVER_PRO:-null}" == "true" ]]; then
     image_name="quay.io/sharelatex/sharelatex-pro"
   fi
+  image_name="${SHARELATEX_IMAGE_REPO:-${image_name}}"
```

Comment:
This would still override `image_name` regardless of `SHARELATEX_IMAGE_REPO` being set or not, I think we'd need a check of existance for `SHARELATEX_IMAGE_REPO`  before overriding `image_name`

Reply:
> Note that image_name=image_name if SHARELATEX_IMAGE_REPO is not set.

That makes sense thanks!